module/VuFind/src/VuFind/Auth/OpenIDConnect.php

```diff
@@ -537,7 +537,12 @@ protected function getAttributeValue(object $userInfo, string $attribute): strin
      * @return array
      * @throws AuthException
      */
-    protected function getJwks(): array
+    * Get signature JWKs from provider
+    *
+    * @return array
+    * @throws AuthException
+    */
+protected function getSignatureJwks(): array
```

Comment:
Looks like some formatting got messed up here:

```suggestion
    protected function getSignatureJwks(): array
```